lib/sync/index.js

```diff
@@ -90,7 +92,7 @@ function invoke(dataset_id, params, callback) {
 
   // Verify that fn param has been passed
   if (!params || !params.fn) {
-    var err = 'no fn parameter provided in params "' + util.inspect(params) + '"';
+    var err = new Error('no fn parameter provided in params "' + util.inspect(params) + '"');
     debugError('[%s] warn %s %j', dataset_id, err, params);
```

Comment:
not sure, but err is no longer string, so debug won't probably show it properly, use %j instead